bigbio/biodatasets/medical_data/medical_data.py

```diff
@@ -146,33 +145,35 @@ def _generate_examples(self, filepath, split: str) -> Tuple[int, Dict]:
                     document["hash"] = line["unique_hash"]
                     document["text"] = line["text"]
                     document["drug_name"] = line["drug"]
-                    document["sentiment"] = line["sentiment"]
+                    document["label"] = line["sentiment"]
                     yield document["hash"], document
             else:
                 for _cols, line in csv_reader.iterrows():
                     document = {}
                     document["hash"] = line["unique_hash"]
                     document["text"] = line["text"]
                     document["drug_name"] = line["drug"]
-                    document["sentiment"] = None
+                    document["label"] = None
                     yield document["hash"], document
 
-        elif self.config.schema == "bigbio_te":
+        elif self.config.schema == "bigbio_pairs":
             csv_reader = pd.read_csv(filepath, dtype="object")
             # Train and test splits handled differently as test split is missing values
             if split == "train":
                 for _cols, line in csv_reader.iterrows():
                     document = {}
                     document["id"] = line["unique_hash"]
-                    document["premise"] = line["text"]
-                    document["hypothesis"] = line["drug"]
+                    document["document_id"] = "NULL"
```

Comment:
if there is no document id provided by the dataset lets make it equal to `id`

Reply:
Done